.github/workflows/ci.yml

```diff
@@ -35,21 +60,24 @@ jobs:
         run: |
           curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
           chmod +x ./cc-test-reporter
-          ./cc-test-reporter before-build        
+      - name: Install dependencies
+        run: bundle install
```

Comment:
I this step needed? If so, why we don't have it for the `linters` job

Reply:
It's not needed. Removed